src/Text/Pandoc/Readers/HTML.hs

```diff
@@ -623,13 +623,49 @@ pPlain = do
      then return mempty
      else return $ B.plain contents
 
-pPara :: PandocMonad m => TagParser m Blocks
-pPara = do
+-- Helper function for pPara when significant attributes are present
+pParaWithWrapper :: PandocMonad m => Attr -> TagParser m Blocks
+pParaWithWrapper (ident, classes, kvs) = do
+  guardEnabled Ext_native_divs -- Ensure native_divs is enabled for this behavior
+  pInhalt <- trimInlines <$> pInTags "p" inline
```

Comment:
I usually use the naming convention of beginning parsers with `p`; so it would be better to use something like `inhalt` instead for this name.

Reply:
Good point! I've renamed it to `contents` to follow your `p` prefix convention for parsers. Thanks for catching that!